src/main/java/com/github/fishio/control/AchievementScreenController.java

```diff
@@ -92,11 +162,56 @@ private void setDefaultOpacities() {
 		smallachieve13.setOpacity(0.1);
 		smallachieve14.setOpacity(0.1);
 		smallachieve15.setOpacity(0.1);
+
 		smallachieve21.setOpacity(0.1);
 		smallachieve22.setOpacity(0.1);
 		smallachieve23.setOpacity(0.1);
 		smallachieve24.setOpacity(0.1);
 		smallachieve25.setOpacity(0.1);
+
+		smallachieve31.setOpacity(0.1);
+		smallachieve32.setOpacity(0.1);
+		smallachieve33.setOpacity(0.1);
+		smallachieve34.setOpacity(0.1);
+		smallachieve35.setOpacity(0.1);
+
+		smallachieve51.setOpacity(0.1);
+		smallachieve52.setOpacity(0.1);
+		smallachieve53.setOpacity(0.1);
+		smallachieve54.setOpacity(0.1);
+		smallachieve55.setOpacity(0.1);
+
+		smallachieve71.setOpacity(0.1);
+		smallachieve72.setOpacity(0.1);
+		smallachieve73.setOpacity(0.1);
+		smallachieve74.setOpacity(0.1);
+		smallachieve75.setOpacity(0.1);
+	}
+
+	/**
+	 * Adds a listener to the Enemy Kill achievement.
+	 */
+	private void addKillListener() {
+		AchievementManager.ENEMY_KILL.getLevelProperty().addListener((o, oVal, nVal) -> {
+			// Fall through is intended.
+			switch (nVal.intValue()) {
+			case 5:
+				smallachieve15.setOpacity(1);
+			case 4:
```

Comment:
*Auto-generated comment*

    Fall through from previous branch of the switch statement.